tests/ui/asm/named-asm-labels.rs

```diff
@@ -171,12 +171,10 @@ fn main() {
     }
 }
 
-// Trigger on naked fns too, even though they can't be inlined, reusing a
-// label or LTO can cause labels to break
```

Comment:
Why is this not a valid concern?

Reply:
I don't think the original reasoning for closing #88169 was correct (even though it did come from me). Fundamentally there's no reason why you can't define global symbols in naked functions, the issue is only with local symbols (`.Lfoo`) which are local to the object file (rustc doesn't guarantee that each asm block is in a separate object file).

In any case #128004 changed naked functions to use `global_asm!` internally so there's no reason to treat it different (this lint is already disabled for `global_asm!`).